### DISCUSSION TITLE
modal component The visible variable replaces the open variable on antd 4.21.6

### DISCUSSION THREAD

**Discussion by mistery-li** [ant-design/ant-design#37424] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/19723639/188585101-ce851bf0-54d3-40c1-882a-42f0be42474c.png)

**Comment by zombieJ** [comment, marked as answer]
`< 4.23.0` 版本是 visible，这个我会重新部署一下官网，把之前的属性也写出来。

**Reply by mistery-li** [reply]
thanks
